packages/subgraph-service/test/unit/subgraphService/indexing-agreement/accept.t.sol

```diff
@@ -226,7 +226,38 @@ contract SubgraphServiceIndexingAgreementAcceptTest is SubgraphServiceIndexingAg
         subgraphService.acceptIndexingAgreement(ctx.indexers[0].allocationId, accepted);
     }
 
-    function test_SubgraphService_AcceptIndexingAgreement_Revert_WhenAgreementAlreadyAllocated() public {}
+    function test_SubgraphService_AcceptIndexingAgreement_Revert_WhenAgreementAlreadyAllocated(
```

Comment:
This test seems unrelated to the fix, right? In which case it would be good to include a test for when an invalid version is supplied